utils/src/main/kotlin/DeclaredLicenseProcessor.kt

```diff
@@ -63,7 +63,7 @@ object DeclaredLicenseProcessor {
     fun process(declaredLicense: String): SpdxExpression? {
         val licenseWithoutPrefixOrSuffix = preprocess(declaredLicense)
```

Comment:
commit-msg: Consider changing the last sentence to explain (shortly) in what way the reporting is actually improved. (As it is not obvious to me what it referrs to)

Reply:
Done.